common/jmeter/oidc/SubOrg_OIDC_AuthCode_Redirect_WithConsent.jmx

```diff
@@ -0,0 +1,1933 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Shall we name the B2B cases starting with "B2B"
Since in future performance testing will highly on b2b cases and it's better if we can quickly categorize them.

Reply:
Addressed with cc345a9a29f0f4f1d7ac39add41d91ccc3176847